pkg/asset/installconfig/azure/validation.go

```diff
@@ -188,11 +192,27 @@ func validateInstanceTypes(client API, ic *types.InstallConfig) field.ErrorList
 		if ultraSSDCapability == "" {
 			ultraSSDCapability = defaultUltraSSDCapability
 		}
+
+		capabilities, err = client.GetVMCapabilities(context.TODO(), instanceType, ic.Azure.Region)
+		if err != nil {
+			allErrs = append(allErrs, field.Invalid(fieldPath.Child("type"), instanceType, err.Error()))
+		}
+
 		if vmNetworkingType == "" {
-			vmNetworkingType = defaultVMNetworkingType
+			val, ok := capabilities[string(azure.AcceleratedNetworkingEnabled)]
+			if ok {
+				if strings.EqualFold(val, "True") {
+					vmNetworkingType = string(azure.VMnetworkingTypeAccelerated)
+				} else {
+					vmNetworkingType = string(azure.VMNetworkingTypeBasic)
+				}
+			}
 		}
 		ultraSSDEnabled := strings.EqualFold(ultraSSDCapability, "Enabled")
-		allErrs = append(allErrs, ValidateInstanceType(client, field.NewPath("controlPlane", "platform", "azure"), ic.Azure.Region, instanceType, diskType, controlPlaneReq, ultraSSDEnabled, vmNetworkingType)...)
+		// if capabilities fails to find a valid instance type, we should skip this validation
+		if capabilities != nil {
+			allErrs = append(allErrs, ValidateInstanceType(client, field.NewPath("controlPlane", "platform", "azure"), ic.Azure.Region, instanceType, diskType, controlPlaneReq, ultraSSDEnabled, vmNetworkingType, capabilities)...)
```

Comment:
Why not pass in the `fieldPath` var you create above?
```suggestion
			allErrs = append(allErrs, ValidateInstanceType(client, fieldPath, ic.Azure.Region, instanceType, diskType, controlPlaneReq, ultraSSDEnabled, vmNetworkingType, capabilities)...)
```